components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/openidconnect/OIDCClaimUtil.java

```diff
@@ -98,6 +98,35 @@ private static String getLocalRoleName(RoleMapping roleMapping) {
         return roleMapping.getLocalRole().getLocalRoleName();
     }
 
+    /**
+     * Filter user claims based on consent with the highest priority {@link OpenIDConnectClaimFilter}. Consent based
+     * user claims filtering can be configured at the global level.
+     */
+    public static Map<String, Object> filterUserClaimsBasedOnConsent(Map<String, Object> userClaims,
```

Comment:
If this method is not recommended to use and is in place just to ensure backward compatibility better if we can deprecate it

Reply:
Better to indicate the method to use instead as well.